scripts/plots/mesoscale/exevs_mesoscale_sref_cnv_plots.sh

```diff
@@ -40,6 +48,7 @@ done
 export init_beg=$first_day
 export valid_beg=$first_day
 
+
```

Comment:
@ShelleyMelchior-NOAA I'm just pointing out extra spaces that I would like removed before this PR is merged. Thanks!